src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -104,7 +131,16 @@ const CityColumnChart = (props) => {
         style: {
           fontSize: '10px',
         },
+        formatter() {
+          // on the city page, show the current city name in bold
+          if (mainCity && this.value === mainCity.name) {
+            return `<span style="font-weight: 700; font-size: 12px">${this.value}</span>`;
```

Comment:
FYI, if you're wondering, I've added 12px/700 to the style guide in my branch (not committed yet) as discussed with Nick.